src/encoded/tests/test_types_init_collections.py

```diff
@@ -1,7 +1,12 @@
+import datetime
 import pytest
 pytestmark = [pytest.mark.setone, pytest.mark.working, pytest.mark.schema]
 
 
+def utc_today_str():
```

Comment:
I suspect we want this in other places as well, might be worth pulling into a different file

Reply:
Yeah, I was thinking it should be in another repo that is shared. Utils or the like.